packages/@aws-cdk/aws-ec2-alpha/lib/route.ts

```diff
@@ -408,7 +441,7 @@ export class RouteTargetType {
 /**
  * Interface to define a route.
  */
-export interface IRouteV2 {
+export interface IRoute extends IConstruct, IResource{
```

Comment:
I see some discrepancy where some changes are from v1 to v2 and some changes are from v2 to v1. Can we make it consistent?

Reply:
i  think its supposed be `IRoute` only, we implemented an idea in between of extending this for introducing one field but went back to the [existing one](https://github.com/aws/aws-cdk/blob/vpcv2-rt-addGateway/packages/%40aws-cdk/aws-ec2-alpha/lib/route.ts#L444C33-L444C54) , this is same as current but i see a nit here `IResource {` that needs to be fixed.
V2 are mainly for the constructs that already exist.